ui/css/logviewer.css

```diff
@@ -53,6 +53,7 @@ body {
 
 .text-danger {
     color: #db0000;
+    background: #fbe3e3 !important;
```

Comment:
I can avoid the `!important` by using more specificity and targeting the child spans, but in doing so I lose the pink treatment on the entire line which looks a bit odd (and probably not so good if you have a really short error). Still looking.

![spanspecificity](https://cloud.githubusercontent.com/assets/3660661/8753583/e4b81578-2c8a-11e5-9480-4b762a5c32b0.jpg)